.github/workflows/ci.yml

```diff
@@ -4,12 +4,12 @@ on: [push, pull_request]
 
 jobs:
   build-macos:
-    runs-on: macos-latest
+    runs-on: macOS-11
     steps:
     # Checks-out the repo. More at: https://github.com/actions/checkout
     - uses: actions/checkout@v2
-    - name: Run on Xcode 11.4
-      run: sudo xcode-select -switch /Applications/Xcode_11.4.app
```

Comment:
`macOS-11` does not support Xcode 11.4, but it does support Xcode 11.7. Close enough.